ztp/remote-worker-day0/ztp-ai/README.md

```diff
@@ -151,4 +151,163 @@ While this approach was tested and worked for UPI deployments, Assisted Installe
+        name: rwn
+        namespace: openshift-ingress-operator
+      spec:
+        replicas: 1
+        domain: apps-rwn.ztp.e2e.bos.redhat.com
+        nodePlacement:
+          nodeSelector:
+            matchLabels:
+              kubernetes.io/hostname: "openshift-worker-2.ztp.e2e.bos.redhat.com"
+        routeSelector:
+          matchLabels:
+            type: rwn-route
+      ~~~
+
+    * Since we need to route traffic to our new domain `apps-rwn.ztp.e2e.bos.redhat.com`, we deployed an external HAProxy that redirects connections to the remote worker node/s where the routers for this domain are running.
+
+    * We labeled our OpenShift Routes targetting apps running on remote worker nodes with the label `type: rwn-route` and then the traffic was handled by the OpenShift router running on the remote worker node and the app was accessed successfully.
+
+**Egress**
+
+* **Egress IP from non-RWN machineNetwork for apps running on RWN:**
+
+  * First, we need to label the RWN with k8s.ovn.org/egress-assignable: ""
+
+    ~~~sh
+    oc label node openshift-worker-2.ztp.e2e.bos.redhat.com k8s.ovn.org/egress-assignable=""
+    ~~~
+
+  * Next we need to create the `EgressIP` configuration:
+
+    ~~~yaml
+    apiVersion: k8s.ovn.org/v1
+    kind: EgressIP
+    metadata:
+      name: egressips-rwn
+    spec:
+      egressIPs:
+      - 10.19.3.50
+      namespaceSelector:
+        matchLabels:
+          type: rwn
+    ~~~
+
+  * We will see that the EgressIP cannot be configured on the remote worker since the node's br-ex interface is not connected to the network where the egressIP lives in.
+
+  * In order to overcome the above issue, we can label a different node (non-rwn) with the `egress-assignable` label, and then we will be able to host that IP somewhere:
+
+    ~~~sh
+    oc label node openshift-worker-0.ztp.e2e.bos.redhat.com k8s.ovn.org/egress-assignable=""
+    ~~~
+
+  * Once we get the EgressIP configured we can run a connection from a pod running on a namespace labeled with the label `type: rwn` and scheduled in the RWN, and we will see we connect using the egressIP:
+
+    > **NOTE**: 10.19.3.46 is an IP from the default machine network. 10.19.142.255 is an IP from the RWN network.
+
+    ~~~sh
+    $ curl http://10.19.3.46:8000
+    $ curl http://10.19.142.255:8000
+    ~~~
+
+    > **NOTE**: These are the logs from the HTTP server we're accessing, we can see source IP 10.19.3.50 (the one configured as egressIP)
+
+    ~~~log
+    10.19.3.50 - - [28/Feb/2022 17:25:43] "GET / HTTP/1.1" 200 -
+    10.19.3.50 - - [28/Feb/2022 17:26:10] "GET / HTTP/1.1" 200 -
+    ~~~
+
+  * The traffic will be routed through the non-RWN node in this case.
+
+* **Egress IP from RWN machineNetwork for apps running on RWN**
+
+  * For this test we need to remove the label we added previously:
+
+    ~~~sh
+    oc label node openshift-worker-0.ztp.e2e.bos.redhat.com k8s.ovn.org/egress-assignable-
+    ~~~
+
+  * And remove the egressIP:
```

Comment:
Not sure if possible, but instead of removing the object, can this be modified/patched? (just for my own curiosity :D, feel free to ignore me)